dev/VSIX/WindowsAppSDK.Extension.sln

```diff
@@ -80,174 +80,356 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WindowsAppSDK.Cs.Extension.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WindowsAppSDK.Cpp.Extension.Dev17", "Extension\Cpp\Dev17\WindowsAppSDK.Cpp.Extension.Dev17.csproj", "{85E88201-049C-4E42-AE85-DFEEFA7C533C}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WinUI.Desktop.Cs.UnitTestApp", "ProjectTemplates\Desktop\CSharp\UnitTestApp\WinUI.Desktop.Cs.UnitTestApp.csproj", "{45777B9F-9CC1-47D7-BD66-C3C194277902}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|arm64 = Debug|arm64
```

Comment:
arm64 should be upper case ARM64. ([it matters](https://github.com/microsoft/WindowsAppSDK-Samples/pull/311/files))

Reply:
Interesting. This is what Visual Studio generated. 

I think I probably want to revert most of these changes. The VSIX is not built for different archs - it only makes sense to build for "Any CPU".

Reply:
sure, if we can chain that into pipelines to map platform appropriately (instead of just passing it along opaquely, which would break).

Reply:
I've removed everything except Any CPU.

Can you explain what your concern is with regards to this breaking something?

In the pipeline yml in the agg repo, I only see this sln being built as Any CPU:
[https://microsoft.visualstudio.com/DefaultCollection/ProjectReunion/_git/WindowsAppSDKAggregator?path=/build/AzurePipelinesTemplates/WindowsAppSDK-CreateVSIX-Steps.yml&version=GBmain&line=37&lineEnd=38&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents](https://microsoft.visualstudio.com/DefaultCollection/ProjectReunion/_git/WindowsAppSDKAggregator?path=/build/AzurePipelinesTemplates/WindowsAppSDK-CreateVSIX-Steps.yml&version=GBmain&line=37&lineEnd=38&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)